src/AzureIoTHub.Portal.Infrastructure/DevelopmentConfigHandler.cs

```diff
@@ -24,6 +24,10 @@ internal DevelopmentConfigHandler(IConfiguration config)
 
         public override string IoTHubConnectionString => this.config[IoTHubConnectionStringKey];
 
+        public override string IoTHubEventHubEndpoint => this.config[IoTHubEventHubEndpointKey];
```

Comment:
Question here. Are we able to discover the IoTHubEventHubEndpoint Url from the IoTHub Registry manager. This might be more safe ?